flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatTreeUI.java

```diff
@@ -570,6 +574,11 @@ protected void paintRow( Graphics g, Rectangle clipBounds, Insets insets, Rectan
 		boolean isSelected = tree.isRowSelected( row );
 		boolean isDropRow = isDropRow( row );
 		boolean needsSelectionPainting = (isSelected || isDropRow) && isPaintSelection();
+		
+		if( alternateRowColor != null && row % 2 != 0 ) {
+			g.setColor( alternateRowColor );
+			FlatUIUtils.paintComponentBackground((Graphics2D) g, 0, bounds.y, tree.getWidth(), bounds.height, 0, 0);
```

Comment:
Use `FlatUIUtils.paintSelection(...)` instead of `FlatUIUtils.paintComponentBackground(...)` for rounded stripes.
See `FlatTreeUI.paintCellBackground()` for how to calculate arc.

Reply:
Sorry, ignore the previous comment regarding the `if( isEditing )` block.